qiskit_ibm_runtime/session.py

```diff
@@ -284,44 +283,28 @@ def service(self) -> QiskitRuntimeService:
     def from_id(
         cls,
         session_id: str,
-        service: QiskitRuntimeService,
+        service: Optional[QiskitRuntimeService] = None,
         backend: Optional[Union[str, IBMBackend]] = None,
     ) -> "Session":
         """Construct a Session object with a given session_id
 
         Args:
-            session_id: the id of the session to be created. This must be an already
```

Comment:
>This must be an already existing session id.

This is actually the correct thing - you can't create a session without submitting a job first as the root job for the session.

Reply:
Will fix in separate PR